src/diagnostics/define_diagnostics.jl

```diff
@@ -726,6 +726,49 @@ function define_diagnostics!(land_model)
             compute_soilco2_source_microbe!(out, Y, p, t, land_model),
     )
 
+    ## Other ##
+    # Longwave out
+    add_diagnostic_variable!(
+        short_name = "lwo",
+        long_name = "Longwave Radiation Out",
+        standard_name = "longwave radiation out",
+        units = "W m^-2",
+        comments = "Longwave radiation going out of the land surface.",
+        compute! = (out, Y, p, t) -> compute_lw_out!(out, Y, p, t, land_model),
+    )
+
+    # Shortwave out
+    add_diagnostic_variable!(
+        short_name = "swo",
+        long_name = "Shortwave Radiation Out",
+        standard_name = "shortwave radiation out",
+        units = "W m^-2",
+        comments = "Shortwave radiation going out of the land surface.",
+        compute! = (out, Y, p, t) -> compute_sw_out!(out, Y, p, t, land_model),
+    )
+
+    # Evapotranspiration
+    add_diagnostic_variable!(
+        short_name = "et",
+        long_name = "Evapotranspiration",
+        standard_name = "evapotranspiration",
+        units = "kg m^-2 s^-1",
+        comments = "Total flux of water mass out of the surface.",
+        compute! = (out, Y, p, t) ->
+            compute_evapotranspiration!(out, Y, p, t, land_model),
+    )
+
+    # Ecosystem respiration
+    add_diagnostic_variable!(
+        short_name = "er",
+        long_name = "Ecosystem Respiration",
+        standard_name = "ecosystem respiration",
+        units = "mol m^-2 s^-1",
+        comments = "Total respiration flux out of the surface.",
+        compute! = (out, Y, p, t) ->
+            compute_total_respiration!(out, Y, p, t, land_model),
```

Comment:
I think we also need a add_diagnostic_variable! for runoff.

and we can make a note that this is Surface runoff only

Reply:
moles of what?

Reply:
@AlexisRenchon